### ISSUE TITLE
Widget icon file doesn't exist

### ISSUE THREAD

**Issue by jdsdev** [nystudio107/craft-retour#37]
Every time the dashboard is loaded there's a warning that the icon file doesn't exist in the logs. Specifically it's looking for this:

```
/vendor/nystudio107/craft-retour/src/assetbundles/retour/dist/img/icon-mask.svg
```

**Comment by khalwat** [comment]
Thanks, will fix!

**Comment by khalwat** [comment]
Fixed in 3.1.6 -> https://github.com/nystudio107/craft-retour/releases/tag/3.1.6
